package.json

```diff
@@ -64,6 +64,14 @@
       "<rootDir>/{client,app,server}/**/*.js",
       "!**/*.{config,test}.js"
     ],
+    "coverageThreshold": {
```

Comment:
What does this achieve?

Reply:
this is the way to configure jest, with a threshold, so if the coverage goes down to this threshold the test will fail

Reply:
ok - so how does this achieve not being able to decrease coverage? Do we have to continually update this ?

Reply:
Yes

Reply:
We can automate the coverage threshold in a separate issue.